### DISCUSSION TITLE
Intranet (LAN) deploy

### DISCUSSION THREAD

**Discussion by zaits07** [lowdefy/lowdefy#1031] [Questions and Help] [answered]
Hello. I excited by lowdefy on my first look. Thaks to developers!

I plan to try to make intranet app with lowdefy.
My server will have internet access in two phases: lowdefy setup and app development.
But server will not have internet access in app using phase.
Is such deploy possible?
Any advices appretiated.
If it possible, could you please show direction?
Any guides or advices how to host lowdefy blocks instead of CDN? Other problems i shoud resolve?

Thanks.

**Comment by SamTolmay** [comment, marked as answer]
Hi @zaits07 

Please have a look at this discussion (#668 ), and if you have further questions, please ask them there to keep the conversation in one place.

We added some features to make hosting your own blocks a little easier, you can look at the "Blocks Server" section of this page in the docs as well: https://docs.lowdefy.com/cli

Unfortunately it is a little complicated to host your own blocks, but this is something that will completely fall away in version 4. We are planning on releasing the first alpha builds early next year.

**Reply by zaits07** [reply]
Big thanks!
